src/themes/heading.js

```diff
@@ -7,34 +7,54 @@ export default {
   level: {
     1: {
       medium: {
-        size: '26px' // 1.625rem
+        size: '26px', // 1.625rem
+        height: 1.2 // (approx)31px
       },
-      large: {
-        size: '32px' // 2rem
+      h1_small: {
```

Comment:
Can we not use underscores here. Please use "h1-small" or 'h1Small" for consistency.

Reply:
Updated to be camel case, thank you!